Tests/SonarScanner.MSBuild.TFS.Test/SummaryReportBuilderTests.cs

```diff
@@ -53,7 +53,7 @@ public void CreateSummaryData_ConfigIsNull_Throws()
         {
             // Arrange
```

Comment:
I suggest creating a [TestInitialize] method and create the logger there and use everywhere in the class from a private field.